packages/fxa-content-server/app/scripts/lib/metrics.js

```diff
@@ -344,7 +353,17 @@ _.extend(Metrics.prototype, Backbone.Events, {
     // reported again.
     this._numStoredAccounts = '';
 
-    this._validateSanitizeUtmParams(filteredData);
+    // Attempt to sanitize the data. Invalid metrics encountered will be reported
+    // individually to sentry and data will be coerced.
+    this._sanitizationErrorCount = 0;
+    filteredData = this.sanitize(filteredData);
+
+    // For now, if any data is coerced and resulted in a sanitization error being
+    // captured do no report the transaction. Depending on the metrics captured in
+    // sentry we may want to revise this strategy.
+    if (this._sanitizationErrorCount > 0) {
```

Comment:
@chenba I just added this. It basically short circuits if we reported sanitization related errors to sentry.